p2p/src/protocol/cbfmgr.rs

```diff
@@ -225,12 +228,15 @@ pub enum GetFiltersError {
 pub struct Config {
     /// How long to wait for a response from a peer.
     pub request_timeout: Timeout,
+    /// Filter cache size, in bytes.
+    pub filter_cache_size: usize,
 }
 
 impl Default for Config {
     fn default() -> Self {
         Self {
             request_timeout: Timeout::from_secs(30),
+            filter_cache_size: 1024 * 1024, // 1 MB.
```

Comment:
Do you think that this size it is better put in the protocol configuration and set 1MB as the default value?

Reply:
That's the plan, this parameter will be passed down from the `protocol::Config` into the `cbfmgr::Config`

Reply:
This is now done!